src/hotspot/cpu/ppc/macroAssembler_ppc.cpp

```diff
@@ -1285,12 +1285,16 @@ bool MacroAssembler::is_load_from_polling_page(int instruction, void* ucontext,
     return true; // No ucontext given. Can't check value of ra. Assume true.
   }
 
-#ifdef LINUX
+#if defined(LINUX) || defined(BSD)
```

Comment:
It seems like there are some mixed uses of defines, but in general I think that in this patch BSD should be _ALLBSD_SOURCE

Reply:
_ALLBSD_SOURCE includes APPLE. Perhaps we should use the new define _BSDONLY_SOURCE which is all the BSD's except APPLE.

Reply:
That's probably moot given that Apple doesn't have any PowerPC hardware.  I don't feel strongly about _ALLBSD_SOURCE vs _BSDONLY_SOURCE here, just that we try to use one of those instead of BSD.

Reply:
I put _ALLBSD_SOURCE